backend/src/main/java/de/neuefische/backend/controller/WatchlistController.java

```diff
@@ -23,9 +23,9 @@ public WatchlistController(WatchlistService watchlistService){
     }
 
     @GetMapping
-    public List<OmdbDetails> getWatchlistByType(Principal principal, @RequestParam Optional<String> type){
+    public List<OmdbDetails> getWatchlistByType(Principal principal, @RequestParam Optional<String> type, @RequestParam String filtered){
```

Comment:
I think the naming could be better, I had to dig through the service code to find out what filtered does. I think this should be boolean, or maybe an enum to better communicate what goes on. Passing in booleans (even as strings 😉 ) as switches to a methodvis a little bit obscure.